ci/dev/vscode.patch

```diff
@@ -31,32 +31,6 @@ index f2ea1bd37010b1eb8a43ce9beaae4a88810f6e2d..3f660f9981921ec465d2b8809a1a5ea5
  	const yarnrc = fs.readFileSync(path.join(REPO_ROOT, 'remote', '.yarnrc'), 'utf8');
  	const target = /^target "(.*)"$/m.exec(yarnrc)[1];
  	return target;
-diff --git a/build/lib/extensions.js b/build/lib/extensions.js
```

Comment:
Something went wrong with the diff here. I think you need to rebase your branch and clean.